lib/node_modules/@stdlib/stats/incr/nancovariance/lib/main.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable stdlib/jsdoc-no-space-aligned-asterisks */
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+
+
+// MAIN //
+
+/**
+ * Returns an accumulator function which incrementally computes a sample covariance.
+ *
+ * @returns {Function} accumulator function
+ *
+ * @example
+ * var accumulator = nancovariance();
+ *
+ * var cov = accumulator();
+ * // returns null
+ *
+ * cov = accumulator( 2.0, 1.0 );
+ * // returns null
+ *
+ * cov = accumulator( 2.0, 5.0 );
+ * // returns 0
+ *
+ * cov = accumulator();
+ * // returns 0
+ */
+function nancovariance() {
+	var meanX = 0.0;
```

Comment:
This implementation is not what we want. The implementation should be a thin wrapper around `incr/covariance`.